oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -260,6 +260,27 @@
                     "are separated with '::'. Example: -Doak.documentstore.persistentCacheIncludes=/content::/var")
     String[] persistentCacheIncludes() default {"/"};
 
+    @AttributeDefinition(
+            name = "Full GC Include Paths",
+            description = "Paths which should be included in full garbage collection." +
+                    "Empty value means all paths are included. " +
+                    "Any path which is added to both include & exclude paths, " +
+                    "would be removed from included paths." +
```

Comment:
Not sure we need to point out someone configuring duplicate paths specifically.

What about making this comment more generic into something like "Include and exclude paths can overlap. Exclude paths will take precedence" ?

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1551/commits/55f310e2d65cfe3208ff7688ce93787eccb0cfba